.ci/bumpStackVersion.groovy

```diff
@@ -47,7 +47,7 @@ pipeline {
   stages {
     stage('Checkout') {
       steps {
-        git(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken', url: "https://github.com/${ORG_NAME}/${REPO}.git")
+        git(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken', url: "https://github.com/${ORG_NAME}/${REPO}.git", branch: "main")
```

Comment:
Should this mean that any repo using this automation should have the `main` branch created?

Reply:
I don't think so. We're explicitly checking out `observability-dev` here: https://github.com/elastic/apm-pipeline-library/pull/1264/files#diff-eba3c46469d30c72b2ce74f5be13c5b7d85e69e0d8ef95dba6e5bf8e3f09a9e8R28